tests/test_split.py

```diff
@@ -454,7 +451,7 @@ def test_get_time_range() -> None:
     events = get_dataset("events")
     query = parse_query(body, events)
 
-    from_date_ast, to_date_ast = _get_time_range(ClickhouseQuery(query), "timestamp")
+    from_date_ast, to_date_ast = get_time_range(ClickhouseQuery(query), "timestamp")
```

Comment:
Shouldn't this test be moved to reflect the new module path for `get_time_range`?

Reply:
right, will move